compiler/src/test/scala/org/scalajs/nscplugin/test/JSSAMTest.scala

```diff
@@ -21,14 +22,22 @@ import org.junit.Test
 
 class JSSAMTest extends DirectTest with TestHelpers {
 
+  override def extraArgs: List[String] = {
+    if (scalaVersion.startsWith("2.11."))
+      super.extraArgs :+ "-Xexperimental"
+    else
+      super.extraArgs
+  }
+
```

Comment:
I'm wondering if it makes sense to put this into a separate commit. IIUC this should probably have been here before already.